change/@internal-react-composites-7452c589-eca7-482e-b144-0572503ede0e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Update lobby screen to reflect to the user who they are calling.",
```

Comment:
the lobby page also handles the 'connecting' state that users land in for when they are joining that call. this page does seem to be pulling double duty for what it is called. we can't change the name of the page right since we expose that state in the API but it seems to be more of a `startingCall` screen or something?